app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCaptureRepositoryImpl.java

```diff
@@ -337,7 +333,7 @@ private ProgramStageSectionRenderingType renderingType(String sectionUid) {
                             }
                         }
                         boolean editable = fieldViewModel.editable() != null ? fieldViewModel.editable() : true;
-                        fieldViewModel = fieldViewModel.withValue(value).withEditMode(editable);
+                        fieldViewModel = fieldViewModel.withValue(value).withEditMode(editable || isEventEditable);
```

Comment:
I won't use an OR operation. If the field is non editable, it should stay like that regardless of the events status. It should be an AND.

Reply:
Okey, I'll check it